Python/Product/Analysis/Analyzer/DDG.cs

```diff
@@ -282,7 +283,7 @@ internal class DDG : PythonWalker {
                         userMod.Imported(_unit);
 
                         foreach (var varName in userMod.GetModuleMemberNames(GlobalScope.InterpreterContext)) {
-                            if (!varName.StartsWith("_")) {
+                            if (!varName.StartsWith("_", StringComparison.Ordinal)) {
```

Comment:
Since we have `StringExtensions`, you can extract `StartsWithOrdinal` method

Reply:
Oh nice idea. This one comes up all the time.